portfolio/src/main/webapp/results.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Results</title>
+    <link rel="stylesheet" href="alexFStyle.css">
+    <script src="script.js"></script>
+  </head>
+  <body onload="getResult()">
+    <div class="main-header">
+        <nav class="nav main-nav">
+            <ul>
+                <li><a href="index.html">HOME</a></li>
+                <li><a href="history.html">HISTORY</a></li>
+            </ul>
+        </nav>
+      <h1 class="header">Results</h1>
+    </div>
+    <div class="content">
+      <p id="result-container" class="result"></p> 
```

Comment:
Minor thing here: would recommend using <div> not <p> for this container like you do for the unlike-container below. That fits a bit more semantically correct I think.